blog/2022-02-07-updating-road-map.md

```diff
@@ -0,0 +1,38 @@
+---
+slug: updating-road-map
+
+title: Updating Roadmap
+
+authors: [jcizmar]
+
+image: "/img/blog/avatar/ogImage.png"
+
+tags: [roadmap, tolgee, future]
+---
+
+In Q4 of 2021 we planned to finish features like translation memory, automated/machine translating, glossaries or paywall 
+for Tolgee cloud. However, we spent a lot of time on upgrading a website and improving JS integrations. Sadly, we 
+haven't finished any of the features, we wanted to finish before end of the year. So we have to reflect it in our
+roadmap. We decided to reschedule everything and target more on the segment of developers than translators.
+Ladies and gentlemen, we are introducing [improved roadmap](/roadmap)! 🎉
+
+## 2022/Q1
+We are just finishing the translation memory, machine translations and automated translating. So users translating their
+projects in Tolgee platform will feel much more comfortable. The platform will be able to suggest translations from 
+the memory and from providers like Goolge Translate or AWS translate.
+
+We would also like to provide more export options, create nice project dashboard with statistics, enable users to see 
+history of translations or add more permission options.
+
+## 2022/Q2
+In the second quarter of 2022 we would like to introduce Glossaries, so user's will be able to manage their terminology 
+and reuse that across multiple projects as well as we would also like to introduce a billing for Tolgee Cloud, so user 
+would be able to buy seats for their team members.
```

Comment:
this paragraph is just one very long phrase. Rewrite it to two shorter phrases so its easier to read and understand.